test/peripheral/521_borrowProxy_base.ts

```diff
@@ -169,21 +171,21 @@ contract('BorrowProxy', async (accounts) => {
           WETH,
           maturity1,
           user2,
+          oneToken.mul(2),
           fyDaiTokens1,
-          oneToken,
           '0x',
           {
             from: user1,
           }
         )
 
-        assert.equal(await dai.balanceOf(user2), oneToken.toString())
+        assert.equal(await dai.balanceOf(user2), oneToken.mul(2).toString())
       })
 
       it("doesn't borrow dai if limit exceeded", async () => {
         await controller.addDelegate(proxy.address, { from: user1 })
         await expectRevert(
-          proxy.borrowDaiForMaximumFYDai(pool.address, WETH, maturity1, user2, fyDaiTokens1, daiTokens1, {
+          proxy.borrowDaiForMaximumFYDai(pool.address, WETH, maturity1, user2, oneToken, 0, {
```

Comment:
```suggestion
          proxy.borrowDaiForMaximumFYDai(pool.address, WETH, maturity1, user2, daiTokens1, fyDaiTokens1, {
```

As above

Reply:
I prefer to address the issue here, so that there isn't an inconsistency between methods in the same contract.

Reply:
Nevermind, extracted to a different PR.